contributor-docs/operations/tests/rfc2119-keyword-test-guide.md

```diff
@@ -0,0 +1,156 @@
+---
+title: SCS RFC2119 Keyword Test Guide
+type:
+status: Draft
+track: Global
+---
+
+## Introduction
+
+The process of test creation and standards definition in Sovereign Cloud Stack (SCS)
+involves developing and validating tests that align with the specific requirements and
+standards set by SCS. This entails a comprehensive understanding of the SCS framework,
+which is built on open-source components and aims to provide interoperable and sovereign
+cloud services. In creating tests for SCS, it's essential to consider the various aspects
+of cloud technology, including infrastructure, services, and the integration of these
+components. The tests should be designed to ensure that the SCS standards are met, which
+includes adhering to specified requirement levels as indicated by [RFC2119 keywords](https://datatracker.ietf.org/doc/html/rfc2119).
```

Comment:
The introduction is good but also very generic. In my opinion, it does not yet answer important questions: Why is it needed/why should I read it? Who should read it (or to whom is it important)?